src/rgw/support/rgw_support_bundle

```diff
@@ -78,7 +77,8 @@ class RGWSupportBundle:
                 if regex.match(file):
                     infile = os.path.join(log_dir, file)
                     outfile = os.path.join(RGWSupportBundle._tmp_src, file)
-                    shutil.copyfile(infile, outfile)
+                    if os.path.isfile(infile):
```

Comment:
this avoids error if any directory matches the regex `regex = re.compile('(rgw*)|(radosgw*)')` in above line number 75

Reply:
this is exactly where CORTX-32068 reported failure exists